src/routes/signin/passwordless/sms/sms.ts

```diff
@@ -91,8 +92,16 @@ export const signInPasswordlessSmsHandler: RequestHandler<
           to: phoneNumber,
         });
     } else {
+      const template = 'signin-passwordless-sms';
```

Comment:
this is somehow a "breaking" change: when an app upgrades to this version, it will use the default template that will render the following sms:
`123456 is your verification code for undefined.`
I would change the default template back to
`Your code is ${code}` and `Votre code est ${code}` in french.

We could then change the default template to your suggestion when we (nhost) will have implemented `APP_NAME` in our console

Reply:
Sorry @plmercereau is this for me?

Do you want me to patch this? When we do not have `APP_NAME` a fallback to be introduced?

Reply:
Or is a general discussion?

Reply:
Applications using the latest released version of hasura-auth have codes like this:
```
Your code is 123456
```

When upgrading with the same configuration, codes will look like this:

```
123456 is your verification code for .
```

Please see the following suggestions

Reply:
Yes I already updated them. Any ideas how we can include APP name to the templates? It will be a breaking change in all cases.